index.js

```diff
@@ -155,15 +166,33 @@ const router = async () => {
                 renderParticipantMessages(participant);
             }
         }
-        else if (route === '#dataCorrectionsTool') {
+        else if (dataCorrectionsToolRoutes.includes(route)) {
             if (JSON.parse(localStorage.getItem("participant")) === null) {
```

Comment:
Is there any need to check for JSON.parse errors here?

Reply:
Sure, but if the participant flag hasn't been set at all, wouldn't this cause an issue with trying to JSON.parse null? (Also, it seems like it's spelled `participant` here, but there's some typos in referencing it elsewhere, noted in my comments.)

Reply:
I believe no issues will be made with `JSON.parse(null)` since it will evaluate to null if the 
key doesn't exist.

Reply:
My understanding is that it will work for `JSON.parse(null)` but not `JSON.parse(undefined)`.

Reply:
I think I understand now, we would also like to handle a situation where `localStorage.setItem('ADD KEY HERE')` is undefined?

Reply:
Understood. I was following the previous conditional checks from the `router` function. This was initially a conditional for the route called `#dataCorrections`. I had it check an array of routes for the conditional.

https://github.com/episphere/studyManagerDashboard/pull/33/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L140-L157